examples/utils/helpers.go

```diff
@@ -0,0 +1,36 @@
+package utils
+
+import (
+	"fmt"
+	"github.com/wasmerio/wasmer-go/wasmer"
+)
+
+func GetGlobal(instance *wasmer.Instance, name string) *wasmer.Global {
+	global, err := instance.Exports.GetGlobal(name)
+
+	if err != nil {
+		panic(fmt.Sprintf("Failed to get the `%s` global: %s\n", name, err))
+	}
+
+	return global
+}
+
+func GetFunction(instance *wasmer.Instance, name string) func(...interface{}) (interface{}, error) {
+	fn, err := instance.Exports.GetFunction(name)
+
+	if err != nil {
+		panic(fmt.Sprintf("Failed to get the `%s` function: %s\n", name, err))
+	}
+
+	return fn
+}
+
+func GetRawFunction(instance *wasmer.Instance, name string) *wasmer.Function {
+	fn, err := instance.Exports.GetRawFunction(name)
+
+	if err != nil {
+		panic(fmt.Sprintf("Failed to get the `%s` function: %s\n", name, err))
+	}
+
+	return fn
+}
```

Comment:
I'm not super fan of this util functions, since it requires an extra dependency on the examples.
I think it might be better to just do this directly on the examples?

PS: perhaps we can actually skip the error checking there.

Reply:
@syrus good point.

What I'll do is remove these helper functions, skip checking for errors in example code and change the notices in the documentation to make it clear that users should (must?) check for errors.